src/v2/Apps/Components/AppContainer.tsx

```diff
@@ -1,17 +1,21 @@
-import { Box, breakpoints } from "@artsy/palette"
+import { Box, BoxProps, useTheme, useThemeConfig } from "@artsy/palette"
+import { ThemeV2, ThemeV3 } from "@artsy/palette/dist/themes"
 import React from "react"
 
-interface AppContainerProps {
-  children: React.ReactNode
-  maxWidth?: number | string
-}
+interface AppContainerProps extends BoxProps {}
 
 export const AppContainer: React.FC<AppContainerProps> = ({
   children,
-  maxWidth = breakpoints.xl,
+  ...rest
 }) => {
+  const {
+    theme: { breakpoints },
+  } = useTheme<ThemeV2 & ThemeV3>()
+
+  const maxWidth = useThemeConfig({ v2: breakpoints.xl, v3: breakpoints.lg })
```

Comment:
I'm failing to parse this... what's happening here?

Reply:
This is a simple hook util that accepts a v2 and v3 key and will return either depending on what version of the theme is active: https://github.com/artsy/palette/blob/1b4bcd9e624523b48078e834cbbef87bc7eabf7c/packages/palette/src/Theme.tsx#L52-L56

It's used a bunch in Palette to return back different values when the theme switches.

So in this case it's choosing between the `xl` and `lg` breakpoints. v3 doesn't have an `xl` size so `lg` is the biggest.

Reply:
Ah, thanks. And that's determined, e.g. in the Gene2 case, by being wrapped in a `<ThemeProviderV3>`, yeah?

Reply:
Exactly, yeah